src/main/java/com/thevoxelbox/voxelsniper/brush/GenerateTreeBrush.java

```diff
@@ -28,7 +28,7 @@
     private Undo undo;
     // If these default values are edited. Remember to change default values in the default preset.
     private Material leavesMaterial = Material.OAK_LEAVES;
-    private Material woodMaterial = Material.OAK_LOG;
+    private Material woodMaterial = Material.OAK_WOOD;
```

Comment:
This could break compatibility with older versions of MC

Reply:
Once again, this works fine for 1.17.